dbfit-java/informix/src/main/java/dbfit/environment/InformixBigDecimalTransformer.java

```diff
@@ -0,0 +1,20 @@
+package dbfit.environment;
+
+import java.sql.SQLException;
+
+import dbfit.util.TypeTransformer;
+
+public class InformixBigDecimalTransformer implements TypeTransformer {
+
+    @Override
+    public Object transform(Object o) throws SQLException {
+        if (o == null) {
+            return null;
+        }
+        if (!(o instanceof dbfit.util.NormalisedBigDecimal)) {
+            throw new UnsupportedOperationException("InformixBigDecimalSpecifier cannot work with " + o.getClass());
```

Comment:
So perhaps we should remove the `throws` clause from the interface's method definition or throw `SQLException` instead (but it's not really database-related, it's just about converting objects between classes).